src/main/java/teammates/common/util/Const.java

```diff
@@ -1138,24 +1138,38 @@ public static final String getjQueryUiFilePath(boolean isDevServer) {
-                + "<br>1. You used a different Google account to access TEAMMATES in the past. "
-                + "In that case, you need to use the same Google account to access TEAMMATES again. "
-                + "Logout and re-login using the other Google account. "
-                + "If you don't remember which Google account you used previously, "
-                + "email us from the same email account to which you receive TEAMMATES emails."
-                + "<br>2. You changed the primary email from a non-Gmail address to a Gmail address recently. "
-                + "In that case, <a href=\"/contact.jsp\">email us</a> "
-                + "so that we can reconfigure your account to use the new Gmail address."
-                + "<br>3. You joined this course just a few seconds ago and your data "
-                + "may be still in the process of propagating through our servers. "
-                + "In that case, please click on the "
-                + "<a href=" + ActionURIs.STUDENT_HOME_PAGE + ">Home</a> link above in a few minutes. "
+        public static final String STUDENT_FIRST_TIME =
+                "<div class=\"align-left\">"
+                    + "<div class=\"align-center text-color-red text-bold\">"
+                        + "Ooops! Your Google account is not known to TEAMMATES"
+                    + "</div>"
+                    + "<br>"
+                    + "To access a course on TEAMMATES, you need to wait till an instructor adds you to "
+                    + "that course. As a confirmation, TEAMMATES will send you instructions on how to access "
+                    + "that particular course."
+                    + "<br><br>"
+                    + "If you 'joined' the course in TEAMMATES using a Google account before, but cannot login "
+                    + "anymore, these are the possible reasons:"
+                    + "<ol>"
+                        + "<li>"
+                            + "You used a different Google account to access TEAMMATES in the past. "
+                            + "In that case, you need to use the same Google account to access TEAMMATES again. "
+                            + "Logout and re-login using the other Google account. "
+                            + "If you don't remember which Google account you used previously, "
+                            + "email us from the same email account to which you receive TEAMMATES emails."
+                        + "</li>"
+                        + "<li>"
+                            + "You changed the primary email from a non-Gmail address to a Gmail address recently. "
+                            + "In that case, <a href=\"/contact.jsp\">email us</a> "
+                            + "so that we can reconfigure your account to use the new Gmail address."
+                        + "</li>"
+                
+                        + "<li>You joined this course just a few seconds ago and your data "
```

Comment:
This part is not standardized in two manners:
- There is an empty line in between `</li>...<li>`, not present previously. (either have an empty line for both, or for none)
- The content after `<li>` here is not line-broken as how it is done for the previous two list items.